### PR TITLE
Addition of twitter feed

### PR DESCRIPTION
**Pull request by dob9601**
Addition of twitter feed to main page along with patch to fix the favicon and org logo

Closes https://github.com/coala/community/issues/40

### CODE REVIEW

**Comment by jayvdb** [comment]
ack https://github.com/coala/community/pull/41/commits/5e6defead4c2cae373ded6160515089832d61b54

**Comment by jayvdb** [comment]
unack https://github.com/coala/community/pull/41/commits/60225f507bb0ef6c76f0a264a193e72c2c359e01

File: gci/views.py [hunk 1]
```diff
@@ -44,8 +44,8 @@ def gci_overview():
         org_logo_file.write(org_logo)
 
     s.append('<link rel="shortcut icon" type="image/png" '
-             'href="static/favicon.png"/>')
-    s.append('<img src="static/org_logo.png" alt="'+org_name+'">')
+             'href="../static/favicon.png"/>')
```
Comment (jayvdb) [inline gci/views.py]: After looking at your patch, I think we need to do the logo as a separate PR, and it should wait until we've done some other improvements to the infrastructure.
I've created an issue for it at https://github.com/coala/community/issues/42

File: community/urls.py [hunk 2]
```diff
@@ -18,7 +19,7 @@ def get_index():
 
 urlpatterns = [
     distill_url(
-        r'^$', TemplateView.as_view(template_name='index.html'),
+        r'^$', overview_index,
```
Comment (jayvdb) [inline community/urls.py]: with this change, the file `templates/index.html` becomes unused.  But we need to be using more raw html & templates, and less code generated HTML (ugly).

Instead of replacing `index.html` I think it would be better for the twitter feed to be on `/twitter/` , in a new app in `/twitter` directory (like activity)

Then the `/twitter` link can be added to  `templates/index.html` .

File: gci/view_overview.py [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+    s.append('<title>Community website</title>')
+    s.append('<ul><li><a href="/gci/">Google Code-in</a>'
+             '<li><a href="/activity/">GitHub activity</a></ul>')
+
+    if api_key_available:
+        api_data_dump = json.loads(
+            requests.get('https://gci-leaders.netlify.com/data.json').content)
+        for item in api_data_dump:
+            if item['name'] == org_name:
```
Comment (jayvdb) [inline gci/view_overview.py]: The new twitter component can load `org_name.txt` which is created at https://github.com/coala/community/blob/master/.ci/build.sh#L5 and https://github.com/coala/community/blob/master/orgname.sh#L5

Then it does not need `org_name = linked_students[0]['organization_name']` and all that mess I wrote last week (I am rewriting it!)

**Comment by jayvdb** [comment]
unack https://github.com/coala/community/pull/41/commits/4eaf47f444cf42cff10d9aa2791be859c763351d

File: gci/view_overview.py (hunk 3 above)
Comment (dob9601) [reply gci/view_overview.py]: that's useful, so the same can be done with the org logo at some stage so that it isn't reliable on api access

**Comment by dob9601** [comment]
@jayvdb how would you want me to access the org_name differently - in the `view_twitter` file i’m accessing it via the `org_name.txt` file.

File: gci/view_twitter.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import requests
+import json
+
+from .gitorg import get_logo
+
+
+def index(request):
+    s = []
+
+    org_name = open('org_name.txt', 'r').read()
```
[the remark this answers is not in the document]
Comment (jayvdb) [reply gci/view_twitter.py]: but `strip()` the contents from the file to remove line feeds etc

File: gci/view_twitter.py [hunk 5]
```diff
@@ -0,0 +1,30 @@
+from django.http import HttpResponse
+import requests
+import json
+
+from .gitorg import get_logo
```
Comment (jayvdb) [inline gci/view_twitter.py]: no logo. twitter only.  dont depend on anything in `gci/`

File: community/urls.py [hunk 6]
```diff
@@ -8,6 +8,7 @@
 from django.views.generic import TemplateView
 
 from gci.views import index as gci_index
+from gci.view_twitter import index as twitter_index
```
Comment (jayvdb) [inline community/urls.py]: this doesnt compile.  `gci.view_twitter` doesnt exist

Comment (jayvdb) [reply community/urls.py]: you can see this on the travis build: https://travis-ci.org/coala/community/builds/316110663

Comment (dob9601) [reply community/urls.py]: sorry, stupid mistake. Will fix it
